_sass/base.scss

```diff
@@ -0,0 +1,98 @@
+.site-header {
+	position: fixed;
+	top: 0;
+	left: 0;
+	width: 100%;
+	border-top: none;
+	border-bottom-color: transparent;
+	background-color: $dark-background-color;
+
+	min-height: 70px;
+	transition: all .5s;
+
+	display: flex;
+	align-items: center;
+
+	&.site-header-scrolled {
+		min-height: 56px;
+	}
+
+	& > .wrapper {
+		width: 100%;
+		position: relative;
+	
+		.site-title {
+			letter-spacing: unset;
+			color: $text-color-light;
+			text-decoration: none;
+		}
+
+		.site-nav .trigger .page-link {
+			color: $text-color-light;
+			margin-left: 1em;
+			margin-right: 1em;
+			@include media-query($on-palm) {
+				color: $text-color;
+			}
+		}
+
+		@include media-query($on-palm) {
+			margin-left: 0;
+			max-width: calc(100vw - 30px);
+	
+			.site-nav {
+				position: absolute;
+				top: 50%;
+				transform: translateY(-18px);
+				right: $spacing-unit/2;
+			}
+		}
+	}
+}
+
+
+.page-content {
+	margin-top: 4em;
+	
+	.post-content {
+		h1 {
+			font-size: 42px;
```

Comment:
My gut instinct is to say that this should be specified as a relative percent, not an absolute quantity

Reply:
Removed it, it's handled by the native styles now